server/devpi_server/importexport.py

```diff
@@ -35,8 +54,8 @@ def do_import(path, xom):
     if not path.check():
         fatal("path for importing not found: %s" %(path))
 
-    with xom.keyfs.transaction():
-        if not xom.model.is_empty():
+    with xom.keyfs.transaction(write=False):
```

Comment:
For my information, why is there `write=False` ?

Reply:
Just wanted to make it explicit that it's a read only transaction. It wouldn't be strictly necessary, because it's the default anyway.